ssc/cmod_levpartflip.cpp

```diff
@@ -884,7 +884,10 @@ enum {
 	CF_battery_replacement_cost_schedule,
 	CF_battery_replacement_cost,
 
-	
+    CF_energy_sales,
+    CF_energy_sales_value,
+    CF_energy_purchases,
+    CF_energy_purchases_value,
```

Comment:
Usually best practice is to add new values to the end of an enumeration. I'm concerned that adding these in the middle might have thrown off some of the cash flow sheets. This is from PVWatts, partnership flip with no debt:

![image](https://user-images.githubusercontent.com/5530592/131411382-5655aed8-186c-4f9e-ab4e-e649af9bb03a.png)

Reply:
At some point the outputs for cf_annual_costs and npv_annual_costs were lost from the var_table. I tried to put them back for FOM at this point but I will need to check all financial models.